common/common.scss

```diff
@@ -481,6 +296,204 @@ hr {
   margin: 0;
   padding-bottom: 1rem;
   padding-top: 0.5rem;
+
+  .row::before {
+    display: none;
+  }
+}
+
+.p-navigation,
+.p-navigation--sliding {
+  .p-navigation-link,
+  .p-navigation__link:hover,
+  .p-navigation__link:visited,
+  .p-navigation__link:focus,
+  .p-navigation__link--search-toggle,
+  .p-navigation__link--search-toggle:hover,
+  .p-navigation__link--search-toggle:visited,
+  .p-navigation__link--search-toggle:focus,
+  .p-navigation__toggle--close,
+  .p-navigation__toggle--close:hover,
+  .p-navigation__toggle--close:visited,
+  .p-navigation__toggle--close:focus,
+  .p-navigation__toggle--open,
+  .p-navigation__toggle--open:hover,
+  .p-navigation__toggle--open:visited,
+  .p-navigation__toggle--open:focus,
+  .p-navigation__dropdown-item,
+  .p-navigation__dropdown-item:hover,
+  .p-navigation__dropdown-item:visited,
+  .p-navigation__dropdown-item:focus {
+    color: #000;
+  }
+}
```

Comment:
This still could be made more concise, `&, &:hover, &:visited, &:focus` pattern is used repeatedly.